cmd/bnsapi/main.go

```diff
@@ -68,13 +68,17 @@ func run(conf Configuration) error {
 	rt := http.NewServeMux()
 	rt.Handle("/info", &handlers.InfoHandler{})
 	rt.Handle("/blocks/", &handlers.BlocksHandler{Bns: bnscli})
+	rt.Handle("/account/nonce/address/", &handlers.NonceAddressHandler{Bns: bnscli})
+	rt.Handle("/account/nonce/pubkey/", &handlers.NoncePubKeyHandler{Bns: bnscli})
 	rt.Handle("/username/owner/", &usernameHandlers.OwnerHandler{Bns: bnscli})
```

Comment:
`curl -X GET "http://localhost:8001/username/owner/04C3DB7CCCACF58EEFCC296FF7AD0F6DB7C2FA17" -H "accept: application/json"` only returns a single value when it should return multiple values:
```
{
  "key": "dG9rZW5zOmRhdmUqaW92",
  "model": {
    "metadata": {
      "schema": 1
    },
    "targets": [
      {
        "blockchain_id": "iov-mainnet",
        "address": "iov1qnpaklxv4n6cam7v99hl0tg0dkmu97sh6007un"
      }
    ],
    "owner": "04C3DB7CCCACF58EEFCC296FF7AD0F6DB7C2FA17"
  }
}
```
`BNSCLI_TM_ADDR=https://rpc.cluster-mainnet.iov.one:443 bnscli query -path '/usernames/owner' -data 04C3DB7CCCACF58EEFCC296FF7AD0F6DB7C2FA17 | jq` yields
```
[
  {
    "Key": "746f6b656e733a646176652a696f76",
    "Value": {
      "metadata": {
        "schema": 1
      },
      "targets": [
        {
          "blockchain_id": "iov-mainnet",
          "address": "iov1qnpaklxv4n6cam7v99hl0tg0dkmu97sh6007un"
        }
      ],
      "owner": "04C3DB7CCCACF58EEFCC296FF7AD0F6DB7C2FA17"
    }
  },
  {
    "Key": "746f6b656e733a65786f6475732a696f76",
    "Value": {
      "metadata": {
        "schema": 1
      },
      "targets": [
        {
          "blockchain_id": "ethereum-eip155-1",
          "address": "0x2c0e0F825E816Fb7997B05c6a6aa0335FD816f67"
        }
      ],
      "owner": "04C3DB7CCCACF58EEFCC296FF7AD0F6DB7C2FA17"
    }
  }
]

```